awx/main/models/events.py

```diff
@@ -546,6 +549,13 @@ def _update_host_summary_from_stats(self, hostnames):
 
             Host.objects.bulk_update(list(updated_hosts), ['last_job_id', 'last_job_host_summary_id'], batch_size=100)
 
+            # bulk-create
+            current_time = now()
+            HostMetric.objects.bulk_create(
+                [HostMetric(hostname=hostname, last_automation=current_time) for hostname in updated_hosts_list], ignore_conflicts=True, batch_size=100
+            )
+            HostMetric.objects.filter(hostname__in=updated_hosts_list).update(last_automation=current_time)
```

Comment:
Answering my own question, there should be as many updates as the number of hosts the job affected, because of the `ignore_conflicts=True` of the bulk_create.... and I think the intent is that these are truly unique on hostname.